modules/configuration/php/NDB_Form_configuration.class.inc

```diff
@@ -106,6 +106,42 @@ class NDB_Form_Configuration extends NDB_Form
         $this->tpl_data['sandbox']         = $config->getSetting("sandbox");
 
     }
+
+    function project() {
+        $factory = NDB_Factory::singleton();
+        $config = $factory->config();
+
+        $projectList = Utility::getprojectList();
```

Comment:
This code should check the config setting useProject before calling Utility:getprojectList().  If useProject is false, getprojectList() will return an error (not just a null array)